Cargo.toml

```diff
@@ -103,7 +102,7 @@ sqlite-federation = ["sqlite"]
 postgres-federation = ["postgres"]
 
 [patch.crates-io]
-datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "dc11bd6edec00fb42ea48b8c5256970c8a533aa2" } # spiceai-47
+datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "e58da1726cfc5b024bad40ff17b7bbd80ca569ab" } # phillip/250521-merge-upstream-df47
```

Comment:
Will update with final commit once https://github.com/spiceai/datafusion-federation/pull/50 is merged